docs/05-gitlab/01-getting-started.mdx

```diff
@@ -22,55 +21,109 @@ Any subsequent steps assume you have read the above.
-   project:
+## Setting Up GitLab CI for Your Unity Project
 
-Assuming you've cloned the example project in the parent folder of your project, and your Unity
-project is at the root of your repository, execute these commands from the root folder of your
-project:
+Follow these steps to quickly set up GitLab CI for your Unity project.
+
+### 1. Run the Commands
+
+Open a terminal in the **parent folder** where you want to set up your Unity project and run these
+commands one by one. In summary, these commands will:
+
+- Clone the example project
+- Copy the necessary files and folders to your Unity project.
+
+:::info
+
+See [example project's release](https://gitlab.com/game-ci/unity3d-gitlab-ci-example/-/releases) for
+stable versions.
+
+:::
+
+Note: This setup assumes that your unity project is at the root of the repository. If your project
+is in a subfolder, you will need to adjust the paths in the commands below. You will also need to
+adjust `UNITY_DIR` in `.gitlab-ci.yml` to point to the correct directory.
 
 ```bash
+# Clone the example repository
+git clone https://gitlab.com/game-ci/unity3d-gitlab-ci-example.git
+
+# Note: you can checkout a specific version if you want
+#cd unity3d-gitlab-ci-example
+#git checkout v4.0.0
+#cd ..
+
+# Change directory to your Unity project folder
+cd your-unity-project
+
+# Create necessary directories
 mkdir -p Assets/Scripts/Editor/
+
+# Copy required files from the example project
 cp ../unity3d-gitlab-ci-example/.gitlab-ci.yml ./
 cp -r ../unity3d-gitlab-ci-example/ci ./
 cp ../unity3d-gitlab-ci-example/Assets/Scripts/Editor/BuildCommand.cs ./Assets/Scripts/Editor/
 ```
 
-1. Open and edit the
-   [`.gitlab-ci.yml`](https://gitlab.com/game-ci/unity3d-gitlab-ci-example/-/blob/main/.gitlab-ci.yml)
-   you copied to your project and update
-   [the variables](https://gitlab.com/game-ci/unity3d-gitlab-ci-example/-/blob/main/.gitlab-ci.yml#L7-13)
-   with the versions you need. Your Unity project version can be found in
-   `ProjectSettings/ProjectVersion.txt`.
-1. If your Unity project is not at the root of your repository, also update UNITY_DIR variable.
-1. Continue to activation instructions
+- Replace `your-unity-project` with the name of your Unity project folder.
+
+<details><summary>Verify the copied files</summary>
+
+After running these commands, you should see the following files and folders in your Unity project
+(in addition to your existing project files):
+
+```plaintext
+your-unity-project/
+├── .gitlab-ci.yml
+├── ci/
+│   ├── nunit-transforms/
+│   │   ├── LICENSE.txt
+│   │   └── nunit3-junit.xslt
+│   ├── before_script.sh
+│   ├── build.sh
+│   ├── docker_build.sh
+│   ├── docker_test.sh
+│   ├── return_license.sh
+│   └── test.sh
+└── Assets/Scripts/Editor/BuildCommand.cs
+```
+
+</details>
+
+### 2. Push Changes to GitLab
+
+Add the files to your repository:
+
+```bash
+git add .gitlab-ci.yml ci/ Assets/Scripts/Editor/BuildCommand.cs
+git commit -m "Add GameCI configuration"
```

Comment:
Nitpick, we could use commitizen style commit message, starting with 'chore: '

Reply:
```suggestion
git commit -m "chore: Add GameCI configuration"
```